core/src/main/scala/org/apache/pekko/persistence/jdbc/state/scaladsl/JdbcDurableStateStore.scala

```diff
@@ -47,7 +47,6 @@ object JdbcDurableStateStore {
 /**
  * API may change
  */
-@ApiMayChange
```

Comment:
The project will not compile on Scala 3 because of this annotation, need to report bug upstream

Reply:
Upstream bug created at https://github.com/lampepfl/dotty/issues/18441